wp-includes/class-wp-web-app-manifest.php

```diff
@@ -61,7 +61,7 @@ public function init() {
 	public function manifest_link_and_meta() {
 		?>
 		<link rel="manifest" href="<?php echo esc_url( rest_url( self::REST_NAMESPACE . self::REST_ROUTE ) ); ?>">
-		<meta name="theme-color" content="<?php echo esc_attr( $this->get_theme_color() ); ?>">
+		<meta name="theme-color" content="<?php echo esc_attr( apply_filters( 'theme_color_meta', $this->get_theme_color() ) ); ?>">
```

Comment:
I should think that this filter would go inside the `get_theme_color` method itself.

Reply:
makes sense... moved.